src/jobs/test.yml

```diff
@@ -36,16 +33,25 @@ parameters:
       The resource class that should be used to run tests. Defaults
       to medium 2 vCPU's and 4GB of RAM. See link for valid inputs:
       https://circleci.com/docs/2.0/configuration-reference/#docker-executor
+
   docker-image:
     type: enum
     enum: [
       "cimg/node@sha256:9e7dff70aee20446a146e374367c3fddc58df1e39cca318dbd4993e625892880",
+      "cimg/node:16.18.1",
```

Comment:
After clark-service is implemented, I think it would be fine if we defaulted to lts